web/src/engine/osk/src/input/gestures/browser/subkeyPopup.ts

```diff
@@ -174,13 +175,19 @@ export default class SubkeyPopup implements GestureHandler {
     const topContainerBounding = topContainer.getBoundingClientRect();
     // Uses the top boundary from `roamBounding` unless the OSK's main element has a more
     // permissive top boundary.
-    const topPadding = Math.min(baseBounding.top + basePadding - topContainerBounding.top, 0);
-    const sustainBounding = new PaddedZoneSource(topContainer, [topPadding, 0, 0])
+    const topPadding = Math.min(baseBounding.top + basePadding * topScalar - topContainerBounding.top, 0);
+    const sustainBounding = new PaddedZoneSource(topContainer, [topPadding * topScalar, 0, 0]);
+
+    let safeBounds = vkbd.gestureEngine.config.safeBounds;
+    if(vkbd.isEmbedded) {
+      safeBounds = new PaddedZoneSource(safeBounds, [topPadding * topScalar, 0, 0]);
+    }
```

Comment:
Just caught this as an issue in testing - going off the top of the iOS or Android keyboard was terminating subkeys and flicks almost immediately.  There will now be much more room before such a thing kicks in.

If we do want that to be fully, 100% disabled... using `NaN`s on line 183 can do the trick.